features/0728-device-binding-attachments/README.md

```diff
@@ -0,0 +1,261 @@
+              },
+              "protected": "eyJhbGciOiJFZERTQSIsImlhdCI6MTU4Mzg4... (bytes omitted)",
+              "signature": "3dZWsuru7QAVFUCtTd0s7uc1peYEijx4eyt5... (bytes omitted)"
+            }
+        }
+    ]
+}
+```
+
+Description of attributes:
+
+- `proofs` -- an array of proofs for different hardware keys which must match the `requests` array from the [device-binding-challenge](#device-binding-challenge)
+    - `id` -- reference to presentation of VC with an embeded hardware bound key 
+    - `path` -- JsonPath to raw value of hardware bound public key within the attached presentation of the VC represented as did:key
+    - `jws` -- Nonce from [device-binding-challenge](#device-binding-challenge) signed with the corresponding private key as a Json Web Signature object
+
+The `device-binding-response` must be attached to the `device_binding~attach` array of a `presentation` message defined by [RFC-0037](https://github.com/hyperledger/aries-rfcs/blob/main/features/0037-present-proof/README.md#presentation) or [RFC-0454](https://github.com/hyperledger/aries-rfcs/tree/main/features/0454-present-proof-v2#presentation).
+
+### Example presentation messages
+
+The following represents a presentation message with an attached libindy presentation and a corresponding device-binding-response.
+
+**Present Proof v1**
+
+```json=
+{
+    "@type": "https://didcomm.org/present-proof/1.0/presentation",
+    "@id": "<uuid-presentation>",
+    "comment": "some comment",
+    "presentations~attach": [
+        {
+            "@id": "libindy-presentation-0",
+            "mime-type": "application/json",
+            "data": {
+                "base64": "<bytes for base64>"
+            }
+        }
+    ], 
+    "device_binding~attach": [
+        {
+            "@id": "device-binding-response-0",
+            "mime-type": "application/json",
+            "data":  {
+                "base64": "<device-binding-response>"
+            }
+        }
+    ]
+}
+```
+
+**Present Proof v2**
+```json=
+{
+    "@type": "https://didcomm.org/present-proof/%VER/presentation",
+    "@id": "<uuid-presentation>",
+    "goal_code": "<goal-code>",
+    "comment": "some comment",
+    "last_presentation": true,
+    "formats" : [
+        {
+            "attach_id" : "libindy-presentation-0",
+            "format" : "hlindy/proof-req@v2.0",
+        }
+    ],
+    "presentations~attach": [
+        {
+            "@id": "libindy-presentation-0",
+            "mime-type": "application/json",
+            "data": {
+                "base64": "<libindy presentation>"
+            }
+        }
+    ],
+    "device_binding~attach": [
+        {
+            "@id": "device-binding-response-0"
+            "mime-type": "application/json",
+            "data":  {
+                "base64": "<device-binding-response>"
+            }
+        }
+    ]
+}
+```
+
+## Reference
+
+
+## Drawbacks
+
+Tbd
```

Comment:
I think we could add a section here about the drawbacks in terms of privacy?

Reply:
done